app/(docs)/node/get-started/install-node-software/cli/dashboard-cli/page.md

```diff
@@ -38,7 +38,7 @@ http://<your-nodes-local-ip>:14002/
 Run the following command to monitor the activity of your node with the CLI dashboard:
 
 ```bash
-docker exec -it storagenode /app/dashboard.sh
+docker exec -it storagenode /app/config/bin/dashboard.sh
```

Comment:
The dashboard script will still be in /app/dashboard.sh. Only the storagenode and storagenode-updater binaries will be moved to /app/config/bin